composer.json

```diff
@@ -0,0 +1,60 @@
+{
+	"name": "nextcloud/welcome",
+	"description": "Dashboard widget to display welcome content",
+	"type": "project",
+	"license": "AGPL-3.0-or-later",
+	"authors": [
+    {
+			"name": "Julien Veyssier",
+			"email": "julien-nc@posteo.net"
+		}
+	],
+	"require": {
+		"php": "^8.0 || ^8.1 || ^8.2",
+		"bamarni/composer-bin-plugin": "^1.8"
+	},
+	"require-dev": {
```

Comment:
Why not adding all dependencies here and having them in vendor-bin?

Reply:
what do you mean? Moving all vendor-bin deps in composer.json directly without the bin support? I think this is a good insurance for the future if anything decides to conflict the core deps.

Reply:
Nevermind, my question was badly formulated.